README.md

```diff
@@ -572,40 +572,8 @@ Go to the [kubernetes-deploy-gem pipeline](https://buildkite.com/shopify/kuberne
 
 # Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/Shopify/kubernetes-deploy.
-
-Contributions to help us support additional resource types or increase the sophistication of our success heuristics for an existing type are especially encouraged! (See tips below)
-
-You can also reach out to us on our slack channel, #krane, at https://kubernetes.slack.com. All are welcome!
```

Comment:
We lost the slack channel reference

Reply:
Good catch. I put it back and also moved the *Development* section to contributing.md